core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -20,10 +20,12 @@ package org.apache.spark
 // scalastyle:off
 import java.io.File
 
-import org.scalatest.{BeforeAndAfterAll, FunSuite, Outcome}
-
 import org.apache.spark.internal.Logging
 import org.apache.spark.util.AccumulatorContext
+import org.scalatest.{BeforeAndAfterAll, FunSuite, Outcome}
+
+import scala.collection.JavaConversions._
```

Comment:
Wonder why the style checker didn't complain, but `scala.*` imports should be in the previous position.

Reply:
I've executed reorganize imports. Shouldn't solve this such problems?

Reply:
I don't know what that is.

The import order is described in http://spark.apache.org/contributing.html, section "Imports".

Reply:
Thanks for the guidance. I've set up the intellij imports organizer as described and fixed with it.